tket/tests/test_TwoQubitCanonical.cpp

```diff
@@ -517,14 +534,20 @@ SCENARIO("KAK Decomposition, various target gate sets") {
     circ.add_op<unsigned>(tket::OpType::PhasedX, {0.23, 0.52}, {1});
     circ.add_op<unsigned>(tket::OpType::CX, {1, 3});
     WHEN("Decomposing to TK2") {
+      Eigen::MatrixXcd u_orig = tket_sim::get_unitary(circ);
       REQUIRE(!Transforms::two_qubit_squash(OpType::TK2).apply(circ));
+      Eigen::MatrixXcd u_res = tket_sim::get_unitary(circ);
```

Comment:
In cases like this (where the transform returns `false`) maybe we should actually check circuit equality rather than unitary equality?

Reply:
Done.